### PR TITLE
Improve outdated documentation

### PR DESCRIPTION
**Pull request by weilbith**
Related to: trustlines-network/project#798

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,16 @@
 # TLBC Monitor
 
-The TLBC Monitor is a program that monitors the Trustlines blockchain and notifies the user about misbehaving or underperforming validators.
+The TLBC Monitor is a program that monitors the Trustlines Blockchain and log
+misbehaving or not well performing validators. The monitor can be easily
+integrated into a complete interactive environment setup with the quickstart
+tool as documented
```
Comment (schmir-at-bb) [inline README.md]: > The monitor can be easily integrated into a complete interactive environment setup with the quickstart

It's a bit unclear what that means. I would probably put something under installation instead:

"The quickstart script provides an easy way to run a tlbc node with tlbc-monitor connected to that. The following instructions describe a manual installation"

File: README.md [hunk 2]
```diff
@@ -61,18 +76,30 @@ In production, the monitor will usually run inside a Docker container and
 interact with another container housing a Parity client connected to the
 Trustlines blockchain.
 
-Here's a list of commands that will start the tlbc-monitoring service in case
-you're running our recommended validator setup:
+Here is a list of commands that will start the tlbc-monitor service in case
+you're running our recommended validator setup. This assumes that you have
+a running Trustlines Blockchain Parity node with the Docker container name
+`tlbc-node`. Please adopt if your setup differs
```
Comment (schmir-at-bb) [inline README.md]: I do think we could get rid of those instructions. The quickstart script (i.e. the recommended validator setup) will already install tlbc-monitor IIRC.

File: README.md (hunk 1 above)
Comment (weilbith) [reply README.md]: :heavy_check_mark:

File: README.md (hunk 2 above)
Comment (weilbith) [reply README.md]: :heavy_check_mark:

**Pull request merged**